osu.Game/Screens/OnlinePlay/Multiplayer/Spectate/MultiSpectatorScreen.cs

```diff
@@ -198,15 +199,29 @@ private bool isCandidateAudioSource(SpectatorPlayerClock? clock)
 
         private void performInitialSeek()
         {
-            // Seek the master clock to the gameplay time.
-            // This is chosen as the first available frame in the players' replays, which matches the seek by each individual SpectatorPlayer.
-            double startTime = instances.Where(i => i.Score != null)
-                                        .SelectMany(i => i.Score.AsNonNull().Replay.Frames)
-                                        .Select(f => f.Time)
-                                        .DefaultIfEmpty(0)
-                                        .Min();
+            // We want to start showing gameplay as soon as possible.
+            // Each client may be in a different place in the beatmap, so we need to do our best to find a common
+            // starting point.
+            //
+            // Preferring a lower value ensures that we don't have some clients stuttering to keep up.
+            List<double> minFrameTimes = new List<double>();
+
+            foreach (var instance in instances)
+            {
+                if (instance.Score == null)
+                    continue;
+
+                minFrameTimes.Add(instance.Score.Replay.Frames.Min(f => f.Time));
+            }
+
+            // Remove any outliers (only need to worry about removing those lower than the mean since we will take a Min() after.
```

Comment:
```suggestion
            // Remove any outliers (only need to worry about removing those lower than the mean since we will take a Min() after).
```